docs/analytics/timeline.md

```diff
@@ -1,15 +1,20 @@
-# Using the Analytics API to assess performance during specific time periods
+# Call Timeline Reports
 
-## Composing a request to the Call Performance Timeline API
+It is quite common to acquire and segment analytics data based on time realted metrics. Call Performance Timeline API provides ability to segment call realted data in various ways such as by getting call duration based on response or actions etc. For more information regarding common use cases, please refer to the [overview section](index.md) and for detailed information regarding how to compose API Request, please refer to the data guide below.
 
-Please refer to our [API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) for details about Timeline API Endpoint.
+## Composing a request using RingCentral API Reference
+
+One can use our [interactive API Reference](https://developers.ringcentral.com/api-reference/Line-Of-Business-Analytics/performanceReportCallsTimeline) to not only create API Request but also run it interactively and see the response.
```

Comment:
Grammar: to not only create **an** API request, but also run it interactively to see the response.

Reply:
fixed